from ..exceptions import FOLError


class NaturalDeductionError(FOLError):
    pass
